python/tvm/topi/arm_cpu/conv2d_gemm.py

```diff
@@ -428,12 +435,29 @@ def schedule_conv2d_gemm_native(cfg, s, out, final_out):
 
     b, m, n = data_im2col.op.axis
     if data_im2col.op.name == "data_im2col":
-        n_outer, n_inner = s[data_im2col].split(n, 16)
+        if A.op.name == "A_padded_K":
```

Comment:
would be good to add a comment here stating that if there are both "padding_K" and "padding_M" ops, this path will be selected